src/libraries/Directory.Build.props

```diff
@@ -96,8 +96,8 @@
     <GenerateDocumentationFile Condition="'$(IsSourceProject)' == 'true' and '$(MSBuildProjectExtension)' != '.vbproj'">true</GenerateDocumentationFile>
     <CLSCompliant Condition="'$(CLSCompliant)' == '' and '$(IsTestProject)' != 'true' and '$(IsTestSupportProject)' != 'true'">true</CLSCompliant>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(IsSourceProject)' == 'true'">
-    <!-- Nullability is enabled by default for all src projects. -->
+  <PropertyGroup Condition="'$(IsSourceProject)' == 'true' or '$(IsReferenceAssemblyProject)' == 'true'">
+    <!-- Nullability is enabled by default for all src and ref projects. -->
     <Nullable Condition="'$(Nullable)' == ''">enable</Nullable>
```

Comment:
Do we only want to set this property if it isn't yet set? Why not set it unconditionally? What if another component like the SDK or Arcade sets that property in the future and with that overrides our setting?

Reply:
I usually unconditionally overwrite properties / items if I want to set a (new) default regardless of what was previously set. In this case it sounds like we want to set our own default. That said, I don't have a strong opinion, so I'm fine leaving it as is if you prefer that.